### PR TITLE
Adds img max-width rule; margin utility classes

### LINKED ISSUE #17: Add max-width for images
This should be in `base.less`:

``` css
img {
    max-width: 100%;
}
```

### LINKED ISSUE #18: Add margin utilities
See: cfpb/cfgov-refresh#154

This is blocking cfpb/cf-layout#1

### PR DESCRIPTION
**Pull request by Scotchester**
- Closes #17.
- Closes #18

### CODE REVIEW

**Comment by himedlooff** [comment]
Gracias, are there other updates to come? This'll need a version bump.

**Comment by Scotchester** [comment]
Whoops, yeah it will, and I just committed stuff for the margin utilities, so I'll batch them together.

**Comment by Scotchester** [comment]
Attn: @ascott1 and @contolini:

This update sets `max-width: 100%;` on all `img` elements. I think it's unlikely this would cause a problem for you, but thought it worth calling to your attention.

File: src/cf-utilities.less [hunk 1]
```diff
@@ -480,6 +480,39 @@
...
+        - .u-m<p>-<#>;
+      notes:
+        - "Replace <p> with the first letter of the position ('t' for top or 'b'
+           for bottom) and <#> with the pixel value of the margin you want."
+        - "Available values: 0, 5, 10, 15, 20, 30, 45, 60."
+  tags:
+    - cf-core
+*/
+
+.u-mt-0  { margin-top:    0; }
```
Comment (himedlooff) [inline src/cf-utilities.less]: If we use the dash between `<p>` and `<#>` we'll have to remember to update our HTML in cfgov-refresh.

Comment (himedlooff) [inline src/cf-utilities.less]: These also need `!important` or else they won't override preset margins.

Comment (Scotchester) [inline src/cf-utilities.less]: I think it's worth it to include the dash, but I don't feel strongly about it. Do you or @virginiacc have a preference?

Comment (Scotchester) [inline src/cf-utilities.less]: Nevermind, I think going dashless is fine. I'll switch to that.

Comment (Scotchester) [inline src/cf-utilities.less]: Updated to include `!important`, too. Thanks for the catch!

**Comment by himedlooff** [comment]
Cool looks good to merge. Should we wait at all for user testing or is that over with now?

**Comment by Scotchester** [comment]
I'm 99% sure it's over, but this won't impact cfgov-refresh yet, anyway.

**Comment by himedlooff** [comment]
Oh right, sorry.

**Comment by Scotchester** [comment]
Thanks!

**Pull request merged**